conf/up4.bess

```diff
@@ -0,0 +1,651 @@
+                    kwargs = None
+                    print('Unable to initialize {} fastpath using alias {},\
+                        falling back to scan'.format(name, pci))
+            if kwargs is None:
+                # Fallback to scanning ports
+                # if port list is empty, scan for dpdk_ports first
+                if not dpdk_ports and scan_dpdk_ports() == False:
+                    print('Registered dpdk ports do not exist.')
+                    sys.exit()
+                # Initialize DPDK fastpath
+                fidx = dpdk_ports.get(mac_by_interface(name))
+                if fidx is None:
+                    raise Exception(
+                        'Registered port for {} not detected!'.format(name))
+                kwargs = {"port_id": fidx, "num_out_q": conf_workers}
+                self.init_fastpath(**kwargs)
+
+            # Initialize kernel slowpath port and RX/TX modules
+            try:
+                peer = peer_by_interface(name)
+                vdev = "net_af_packet{},iface={}".format(idx, peer)
+                slow = PMDPort(name="{}Slow".format(name), vdev=vdev)
+                spi = __bess_module__("{}SlowPI".format(name), 'PortInc', port=slow.name)
+                spo = __bess_module__("{}SlowPO".format(name), 'PortOut', port=slow.name)
+                qspo = __bess_module__("{}QSlowPO".format(name), 'Queue')
+
+                # host_ip_filter: tcpdump -i foo 'dst host 198.19.0.1 or 198.18.0.1' -d
+                # Should always be set to lowest priority
+                HostGate = MAX_GATES - 1
+                ips = ips_by_interface(name)
+                host_ip_filter = {"priority": -HostGate, "filter": "dst host "
+                                + " or ".join(str(x) for x in ips), "gate": HostGate}
+
+                self.bpf.add(filters=[host_ip_filter])
+
+                # Direct control traffic from DPDK to kernel
+                self.bpf:HostGate -> qspo -> spo
+
+                # Direct control traffic from kernel to DPDK
+                spi -> self.fpo
+
+                tc = 'slow{}'.format(wid)
+                try:
+                    bess.add_tc(tc, policy='round_robin', wid=wid)
+                except Exception as e:
+                    if e.errmsg == "Name '{}' already exists".format(tc):
+                        pass
+                    else:
+                        raise e
+                # Limit scheduling slow path RX/TX to 1000 times/second each
+                for mod in spi, qspo:
+                    bess.add_tc(mod.name,
+                            parent=tc,
+                            policy='rate_limit',
+                            resource='count',
+                            limit={'count': 1000})
+                    mod.attach_task(mod.name)
+            except Exception as e:
+                print('Mirror veth interface: {} misconfigured: {}'.format(name, e))
+        else:
+            raise Exception('Invalid mode selected.')
+
+
```

Comment:
Lets move this section into a helper file too, like the parser

Reply:
https://github.com/omec-project/upf-epc/pull/54#discussion_r432986888